geomesa-arrow/geomesa-arrow-gt/src/main/scala/org/locationtech/geomesa/arrow/io/DictionaryBuildingWriter.scala

```diff
@@ -106,7 +108,12 @@ class DictionaryBuildingWriter(
 
     val dictionaries = attributeWriters.collect { case w: ArrowDictionaryWriter =>
       val name = s"dict-${w.dictionary.encoding.getId}"
-      val descriptor = SimpleFeatureTypes.renameDescriptor(sft.getDescriptor(w.vector.getField.getName), name)
+      val original = sft.getDescriptor(w.vector.getField.getName)
+      val descriptor = if (original.isList) {
```

Comment:
Does this logic happen enough that it'd make sense to extract an ArrowSimpleFeatureTypes helper thingy?

Reply:
yeah, maybe... it's fairly simple and as I said in the other comment most places were slightly different. I could go either way, what do you think?

Reply:
Hmm....  I'm fine either way.  It isn't the end of the world if things are a little WET.  Just wanted to ask about it.